killunit.js

```diff
@@ -0,0 +1,57 @@
+// totally original, not stolen from a better programmer
+(function(){
+    const args = parseArguments(argument);
+
+    if (args.length === 0) return "specify a unit cheesehead";
+
+    let newGamemode = args[0].length === 1 ? args[0] : args[0].slice(0, 2)
+    switch (newGamemode.toLowerCase()) {
```

Comment:
```suggestion
    switch (args[0].toLowerCase()) {
```
> `args[0].length === 1 ? args[0] : args[0].slice(0, 2)`

that would return the first 2 or first charactor of the argument